src/FixedDataTable.js

```diff
@@ -920,7 +920,7 @@ var FixedDataTable = React.createClass({
     }
 
     var lastScrollTop = oldState ? oldState.scrollTop : undefined;
-    if (props.scrollTop !== lastScrollTop) {
+    if (typeof props.scrollTop !== 'undefined' && props.scrollTop !== null && props.scrollTop !== lastScrollTop) {
```

Comment:
Hmm that isn't my finding here: http://jsfiddle.net/s14sav6k/4/
But I'm not too familiar with strict mode.  Do you have documentation which describes what can cause the ReferenceErrors?  Could it be a browser specific implementation?